Tools/clinic/clinic.py

```diff
@@ -1797,7 +1797,7 @@ def is_stop_line(line):
             for field in shlex.split(arguments):
                 name, equals, value = field.partition('=')
                 if not equals:
-                    fail("Mangled Argument Clinic marker line: {!r}".format(line))
+                    fail(f"Mangled Argument Clinic marker line: {line!r}")
```

Comment:
```suggestion
                    fail("Mangled Argument Clinic marker line:", line)
```

(I'm pretty sure `line` is always a string here.)

Reply:
Yes, but the `repr()` call still has an impact on the error message (and a positive one, in my opinion):

```pycon
>>> line = 'this is the line'
>>> print('line:', line)
line: this is the line
>>> print('line:', repr(line))
line: 'this is the line'
```

Reply:
Doh, of course 🤦

Reply:
```suggestion
                    fail("Mangled Argument Clinic marker line:", repr(line))
```